api/catalog/api/serializers/media_serializers.py

```diff
@@ -277,7 +277,7 @@ class TagSerializer(serializers.Serializer):
         help_text="The name of a detailed tag.",
     )
     accuracy = serializers.FloatField(
-        required=False,
+        default="0.0",
```

Comment:
Setting accuracy to a default of zero suggests that the tags are completely inaccurrate! Setting it to `None` has the intended meaning that we are uncertain about the accuracy.

```suggestion
        default=None,
```

Reply:
Thank you dhruv, will do the same and revert as soon as possible

Reply:
Done